services/rds/rds.go

```diff
@@ -302,7 +305,21 @@ func (svc *Service) Discover(ctx context.Context, accessKey, secretKey string) (
 					return errors.WithStack(err)
 				}
 
-				out, err := rds.New(s).DescribeDBInstancesWithContext(ctx, new(rds.DescribeDBInstancesInput))
+				pageNum := 0
+				DBInstanceOutput := []*rds.DescribeDBInstancesOutput{}
+
+				err = rds.New(s).DescribeDBInstancesPagesWithContext(ctx, new(rds.DescribeDBInstancesInput),
```

Comment:
We should use `maxPageNum` in `DescribeDBInstancesInput`.

Reply:
page -> pagination